prometheus/Dockerfile

```diff
@@ -7,7 +7,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Setup base system
 ARG BUILD_ARCH=amd64
-ENV PROMETHEUS_VERSION=2.36.1
+ENV PROMETHEUS_VERSION=2.37.0
```

Comment:
This should be in a separate PR.

Reply:
Ok, I will resubmit at the 2.36.1 version. But since 2.37.0 is the LTS version of Prometheus it would be good to use that one.